src/components/table/template.njk

```diff
@@ -0,0 +1,27 @@
+<table class="govuk-c-table {{ classes }}">
+  {% if caption %}
+  <caption class="govuk-c-table__caption {{captionSize}}">{{ caption }}</caption>
+  {% endif %}
+  {% if data.head %}
+  <thead class="govuk-c-table__head">
+    <tr class="govuk-c-table__row">
+    {% for label, type in data.head %}
```

Comment:
- `label` doesn't seem like the right name here when it could be e.g. `'£75.00'`.
- what you're referring to as `type` is currently either an empty hash or a hash containing {'type': 'numeric'}. Would it make more sense to call this `options`?